.grabl/automation.yml

```diff
@@ -0,0 +1,148 @@
+#
+#
+
+build:
+  quality:
+    filter:
+      owner: graknlabs
+      branch: master
+    job:
+      build-analysis:
+        machine: graknlabs-ubuntu-20.04
+        script: |
+          SONARCLOUD_CODE_ANALYSIS_CREDENTIAL=$SONARCLOUD_CREDENTIAL \
+            bazel run @graknlabs_dependencies//tool/sonarcloud:code-analysis -- \
+            --project-key=graknlabs_common \
+            --branch=$GRABL_BRANCH --commit-id=$GRABL_COMMIT
+  correctness:
+    jobs:
+      build:
+        machine: graknlabs-ubuntu-20.04
+        script: |
+          bazel build --config=rbe //... --test_output=errors
+          bazel run @graknlabs_dependencies//tool/checkstyle:test-coverage
+          bazel test --config=rbe $(bazel query 'kind(checkstyle_test, //...)')
+      build-dependency:
+        machine: graknlabs-ubuntu-20.04
+        script: |
+          dependencies/maven/update.sh
+          git diff --exit-code dependencies/maven/artifacts.snapshot
+          bazel run @graknlabs_dependencies//tool/unuseddeps:unused-deps -- list
+      deploy-maven-snapshot:
+        filter:
+          owner: graknlabs
+          branch: master
+        machine: graknlabs-ubuntu-20.04
+        script: |
+          export DEPLOY_MAVEN_USERNAME=$REPO_GRAKN_USERNAME
+          export DEPLOY_MAVEN_PASSWORD=$REPO_GRAKN_PASSWORD
+          bazel run --define version=$(git rev-parse HEAD) //:deploy-maven -- snapshot
+      deploy-apt-snapshot:
+        filter:
+          owner: graknlabs
+          branch: master
+        machine: graknlabs-ubuntu-20.04
+        script: |
+          export DEPLOY_APT_USERNAME=$REPO_GRAKN_USERNAME
+          export DEPLOY_APT_PASSWORD=$REPO_GRAKN_PASSWORD
+          bazel run --define version=$(git rev-parse HEAD) //binary:deploy-apt -- snapshot
+      deploy-rpm-snapshot:
+        filter:
+          owner: graknlabs
+          branch: master
+        machine: graknlabs-ubuntu-20.04
+        script: |
+          sudo apt-get update && sudo apt-get install rpm
+          export DEPLOY_RPM_USERNAME=$REPO_GRAKN_USERNAME
+          export DEPLOY_RPM_PASSWORD=$REPO_GRAKN_PASSWORD
+          bazel run --define version=$(git rev-parse HEAD) //binary:deploy-rpm -- snapshot
+      test-deployment-maven:
+        filter:
+          owner: graknlabs
+          branch: master
+        machine: graknlabs-ubuntu-20.04
+        script: |
+          sudo apt-get update
+          sudo apt-get -y install maven
+          export ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME
+          export ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD
+          bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
+          bazel run //test:grakn-extractor -- dist/grakn-core-all-linux
+          ./dist/grakn-core-all-linux/grakn server &
+          sed -i -e "s/CLIENT_JAVA_VERSION_MARKER/$GRABL_COMMIT/g" test/deployment/pom.xml
+          cat test/deployment/pom.xml
+          cd test/deployment && mvn test
```

Comment:
This job is not applicable for this repo, @alexjpwalker . We don't have a `//test/deployment` package in this repo, and there's no deployment test to be done. We also don't have `//test:grakn-extractor` defined either. We should read the code carefully before copying them over, as it would have told the whole story rather clearly. ;)

Reply:
That was careless, sorry